pkg/artifactory/provider.go

```diff
@@ -23,6 +23,7 @@ func Provider() *schema.Provider {
 		"artifactory_local_nuget_repository":      resourceArtifactoryLocalNugetRepository(),
 		"artifactory_local_maven_repository":      resourceArtifactoryLocalJavaRepository("maven", false),
 		"artifactory_local_alpine_repository":     resourceArtifactoryLocalAlpineRepository(),
+		"artifactory_local_cargo_repository":      resourceArtifactoryLocalCargoRepository(),
```

Comment:
No conda here

Reply:
`conda` is added to the list of generic repos. So it will be automatically generated.